models/cost_per_query.sql

```diff
@@ -56,25 +69,34 @@ query_seconds_per_hour as (
         datediff('millisecond', greatest(execution_start_time, hour_start), least(end_time, hour_end)) as num_milliseconds_query_ran,
         sum(num_milliseconds_query_ran) over (partition by warehouse_id, hour_start) as total_query_milliseconds_in_hour,
         div0(num_milliseconds_query_ran, total_query_milliseconds_in_hour) as fraction_of_total_query_time_in_hour,
+        sum(query_acceleration_bytes_scanned) over (partition by warehouse_id, hour_start) as total_query_acceleration_bytes_scanned_in_hour,
+        div0(query_acceleration_bytes_scanned, total_query_acceleration_bytes_scanned_in_hour) as fraction_of_total_query_acceleration_bytes_scanned_in_hour
```

Comment:
@NiallRees would appreciate your thoughts on this.

To distribute query_acceleration costs, we only have two options:

1. allocate them proportional partitions scanned
2. allocate them proportional to bytes scanned 

Neither option is great, you ultimately need the extra compute used by the query. Went with bytes scanned since it will be more correlated with actual extra compute used compared to partitions scanned.


(the only fields we get in query history for QA are:

    query_acceleration_bytes_scanned,
    query_acceleration_partitions_scanned,
    query_acceleration_upper_limit_scale_factor

)

Reply:
Sounds like the best option available 👍